manifests/lsstsw.pp

```diff
@@ -1,17 +1,20 @@
 # == Define: lsststack::lsstsw
 #
 define lsststack::lsstsw(
-  $user            = $title,
-  $group           = $title,
-  $manage_user     = true,
-  $manage_group    = true,
-  $lsstsw_repo     = 'https://github.com/lsst/lsstsw.git',
-  $lsstsw_branch   = 'master',
-  $lsstsw_ensure   = 'present',
-  $buildbot_repo   = 'https://github.com/lsst-sqre/buildbot-scripts.git',
-  $buildbot_branch = 'master',
-  $buildbot_ensure = 'present',
-  $debug           = false,
+  $user              = $title,
```

Comment:
I have mixed feelings about but it's idiomatic puppet.  The linter doesn't warn on this case (yet) but it will complain about unaligned hash rockets.